src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/">Home</Link>
         </div>
         <div>
-          <a href="">About</a>
+          <Link to="about">About</Link>
```

Comment:
Be careful with these links, as they will just add `about` to the end of the current url path. We generally want them to match the route, so they should be `/about` and `/contact`